packages/tools/kadena-cli/src/account/utils/accountHelpers.ts

```diff
@@ -9,6 +9,18 @@ import { services } from '../../services/index.js';
 import { notEmpty } from '../../utils/helpers.js';
 import { isEmpty } from './addHelpers.js';
 
+export const accountResolveAllowedNetworks = [
```

Comment:
I'm not so sure about this, if we hard-code this what is the point of having user-defined networks? A user could add their own network that allows us te resolve accounts and everything.